ee/indexeddb/comparer.go

```diff
@@ -0,0 +1,33 @@
+package indexeddb
+
+import (
+	"github.com/kolide/goleveldb/leveldb/comparer"
+)
+
+// chromeComparer is a thin wrapper around the default comparer to allow us
+// to use it as `idb_cmp1`.
```

Comment:
Should `idb_cmp1` be meaningful to a reviewer?

Reply:
Ah nope -- it is indexeddb-specific. I will improve the documentation here!